rootbeer/constant-source/test-constant-source.html

```diff
@@ -0,0 +1,183 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Rootbeer Test: ConstantSource</title>
+    <script src="../resources/testharness.js"></script>
+    <script src="../resources/testharnessreport.js"></script>
+    <script src="../resources/audit.js"></script>
+  </head>
+
+  <body>
+    <script>
+      const audit = Audit.createTaskRunner();
+
+      let processorPath = "../processors/constant-source-processor.js";
+
+      let context;
+
+      class RB_ConstantSourceNode extends AudioWorkletNode {
+        constructor(context, options) {
+          let procOptions = {
+            numberOfinputs : 0,
+            numberOfOutputs : 1,
+            parameterData : {offset : (options && options.offset) || 1}
+          };
+
+          console.log(`Node options`);
```

Comment:
Why `console.log()`? Are we using this for testing?

Reply:
Ditto for all `console.log()` instances below. I suggest to wrap this into an utility function so you can disable them easily.